samples/msal-browser-samples/VanillaJSTestApp2.0/app/client-capabilities/test/browser.spec.ts

```diff
@@ -27,6 +36,16 @@ describe("Browser tests", function () {
         const labClient = new LabClient();
         const envResponse = await labClient.getVarsByCloudEnvironment(labApiParams);
         [username, accountPwd] = await setupCredentials(envResponse[0], labClient);
+        [clientID, , authority] = await retrieveAppConfiguration(envResponse[0], labClient, false);
```

Comment:
Rather than a `[clientId, , authority]`  suggest you rewrite this to something more like `[clientID, unused, authority]` to improve readability.  You have this pattern in several places in this PR.

Reply:
ah, fair! -will replace them all

Reply:
@peterzenz well it turned out to be a little more tricky -since we are doing initalization/reassignment here, I would have to define `unused` somewhere above (which also feels weird). I'd like to refactor `retrieveAppConfiguration` to return an object instead of array, and match the lab api [schema](https://msidlab.com/swagger/v1/swagger.json) a bit more.